plugins/modules/fabric_devices_workflow_manager.py

```diff
@@ -0,0 +1,4080 @@
+                        - If remote_ipv6_address is provided, then local_ipv6_address is mandatory.
+                        - If both are set, external_connectivity_ip_pool_name will be given higher priority.
+                        type: str
+                      remote_ipv6_address:
+                        description:
+                        - IP address of a device located outside the fabric network, often used for BGP peering.
+                        - remote_ipv6_address is for IPv6.
+                        - local_ipv6_address and remote_ipv6_address should fall in same subnet.
+                        - If local_ipv6_address is provided, then local_ipv6_address is mandatory.
+                        - If both are set, external_connectivity_ip_pool_name will be given higher priority.
+                        type: str
+                  layer3_handoff_sda_transit:
+                    description:
+                    - Adds layer 3 handoffs with SDA transit in fabric devices.
+                    - Configured when routing traffic is routed from the SDA fabric to external networks.
+                    - If layer3_handoff_sda_transit is set, border type will be considered as Layer3.
+                    type: dict
+                    suboptions:
+                      transit_network_name:
+                        description:
+                        - Network that connects multiple SDA fabrics or networks.
+                        - Mandatory parameter for all operations in L3 Handoff with SDA transit.
+                        - transit_network_name cannot be updated.
+                        type: str
+                      affinity_id_prime:
+                        description:
+                        - It supersedes the border priority to determine border node preference.
+                        - The lower the relative value of affinity id prime, the higher the preference.
+                        - Resources with the same affinity ID are treated similarly and affinity_id_decider decides the priority.
+                        - affinity_id_prime ranges from 0 to 2147483647.
+                        - affinity_id_prime can be updated.
+                        type: int
+                      affinity_id_decider:
+                        description:
+                        - If affinity id prime value is the same, the affinity id decider value is used as a tiebreaker.
+                        - The lower the relative value of affinity id prime, the higher the preference.
+                        - affinity_id_decider ranges from 0 to 2147483647.
+                        - affinity_id_decider can be updated.
+                        type: int
+                      connected_to_internet:
+                        description:
+                        - Set this true to allow associated site to provide internet access to other sites through SDA.
+                        - Default value is False.
```

Comment:
false

                        - This parameter can be updated.

default: false